ols/app/models/config.py

```diff
@@ -219,31 +219,35 @@ def validate_yaml(self) -> None:
             v.validate_yaml()
 
 
-class RedisCredentials(BaseModel):
-    """Redis credentials."""
+class RedisTLSCerts(BaseModel):
+    """Redis TLS certificates."""
 
-    username: Optional[str] = None
-    password: Optional[str] = None
+    cert_path: Optional[str] = None
+    key_path: Optional[str] = None
 
     def __init__(self, data: Optional[dict] = None) -> None:
-        """Initialize redis credentials."""
+        """Initialize redis TLS certificates."""
         super().__init__()
         if not isinstance(data, dict):
             return
-        self.username = _get_attribute_from_file(data, "username_path")
-        self.password = _get_attribute_from_file(data, "password_path")
+        self.cert_path = data.get("cert_path")
+        self.key_path = data.get("key_path")
```

Comment:
why does OLS need the redis tls key?  it should only need to be able to verify the cert, and it should be able to verify the cert just based on it being signed by the OCP CA.  So it shouldn't need either of these things to be able to make a trusted TLS connection to Redis.

Reply:
Sorry this was one my `no-auth` experiments that i have accidentally committed. Yes we don't require TLS config in OLS. OLS as a client just needs to have a CA cert with it to trust the identity of the redis server and talk to it. Redis server is configured with TLS cert, key and CA cert. Certificate verification has been made mandatory to make sure the communication is encrypted between them.